default_repos.yaml

```diff
@@ -10,7 +10,7 @@ repositories:
   mirte-install-scripts:
     type: git
     url: https://github.com/mirte-robot/mirte-install-scripts.git
-    version: main
+    version: v0.1.1-dev
```

Comment:
This should remain main? v0.1.1-dev of the install scripts just got merged

Reply:
Indeed shouldn't be in there.